src/Workspaces/Core/Portable/Workspace/IsolatedAnalyzerReferenceSet.Core.cs

```diff
@@ -184,6 +184,11 @@ public static async partial ValueTask<ImmutableArray<AnalyzerReference>> CreateI
         SolutionServices solutionServices,
         CancellationToken cancellationToken)
     {
+        // Fallback to stock behavior if the reloading option is disabled.
+        var optionsService = solutionServices.GetRequiredService<IWorkspaceConfigurationService>();
```

Comment:
Is this check required? Looks like it's covered by the other method, no?

Reply:
There are two public entrypoints.  Both need the check.